src/main/java/de/unijena/cheminf/clustering/desccalc/Descriptor.java

```diff
@@ -1106,17 +1134,7 @@ void calculate(IAtomContainer anAtomContainer, float[] aVector, int aStartIndex)
     PUBCHEM_FINGERPRINTER(true, true, true, false, 881) {
         @Override
         void calculate(IAtomContainer anAtomContainer, float[] aVector, int aStartIndex) {
```

Comment:
(I'm marking this here because I cannot comment above) [nitpick] Here, it is not really "aStartIndex" like in the method above, where it really defines from which matrix column on, the descriptors should be entered, but rather the specific index where this specific descriptor should be entered into the matrix. I would suggest changing the param name here simply to "index".